.github/workflows/storybook.yml

```
@@ -0,0 +1,40 @@
+name: Build and Deploy Storybook to Chromatic
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+  build_and_deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Install Dependencies
+        run: npm install
+        working-directory: ./ui
+
+      - name: Build Storybook
+        run: npm run build-storybook
+        working-directory: ./ui
+
+      - name: Publish to Chromatic
+        uses: chromaui/action@v1
+        with:
+          token: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
```

```suggestion
          token: ${{ secrets.GITHUB_TOKEN }}
```

I think this is the GH token, not the Chromatic one